modules/nw-aws-switching-clb-with-nlb.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-aws.adoc
+
+:_content-type: PROCEDURE
+[id="nw-aws-switching-clb-with-nlb_{context}"]
+= Switching the Ingress Controller from using a Classic Load Balancer to a Network Load Balancer
+
+You can switch the Ingress Controller that is using a Classic Load Balancer (CLB) to one that uses a Network Load Balancer (NLB) on AWS.
+
+Switching between these load balancers will not delete the IngressController object and thus does not cause an Ingress interruption.
+
+.Procedure
+
+. Modify the existing Ingress Controller that you would like to switch to using a NLB. The following example assumes that your default Ingress Controller has an `External` scope and no other customizations:
++
+.Example `ingresscontroller.yml` file
```

Comment:
Most documentation uses `.yaml`, but `.yml` is used here and some other places.  Either should work, and the `oc apply` command doesn't care how the file is named (as long as the actual file name is provided to `oc apply`), but the inconsistency caught my eye.

Reply:
Good catch, for consistency I'll update this to use the `.yaml` extension